ackermann_steering_controller/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ackermann_steering_controller)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  diff_drive_controller
+  hardware_interface
+  nav_msgs
+  pluginlib
+  realtime_tools
+  roscpp
+  tf
+  urdf
+)
+
+find_package(Boost REQUIRED COMPONENTS system thread)
+ 
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  CATKIN_DEPENDS
+    controller_interface
+    diff_drive_controller
+    hardware_interface
+    nav_msgs
+    realtime_tools
+    roscpp
+    tf
+  DEPENDS Boost
+)
+
+include_directories(
+  include
+  ${Boost_INCLUDE_DIRS} 
+  ${catkin_INCLUDE_DIRS}
+)
+
+list(APPEND CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
This line can now get removed.
(In general CMAKE_* variables should not get touched at all..)

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/pull/356/commits/957b22f3928e183d567db30de70a08d0575cfa05